src/common/urlUtils.ts

```diff
@@ -23,6 +25,53 @@ export function getCaseSensitivePaths() {
   return isCaseSensitive;
 }
 
+/**
+ * Lowercases the path if the filesystem is case-insensitive. Warning: this
+ * should only be done for the purposes of comparing paths. Paths returned
+ * through DAP and other protocols should be correctly-cased to avoid incorrect
+ * disambiguation.
+ */
+export function lowerCaseInsensitivePath(path: string) {
+  return isCaseSensitive ? path : path.toLowerCase();
+}
+
+/**
+ * Returns the path in its true case, correcting for case-insensitive file systems.
+ */
+export const truePathCasing = memoize(
```

Comment:
Not used anywhere yet right?

Reply:
It was used in the repo but we don't need to any more.